docs/Actions.md

```diff
@@ -774,6 +774,27 @@ appetize(
 )
 ```
 
+### [appaloosa](https://www.appaloosa-store.com)
+​
+Upload your ipa or apk to your private store on Appaloosa.
+​
+Add the `appaloosa` action after the `ipa` step or use it with your existing `apk`.
+​
+You can add some options:
+```ruby
+appaloosa(
+  binary: '/path/to/binary.ipa', # path tor your IPA or APK
+  store_id: 'your_store_id', # you'll be asked for your email if you are not already registered 
+  api_token: 'your_api_key', # only if already registered
+  group_ids: '1, 2, 3', # User groups visibility, default: 'everybody'
```

Comment:
This is the ID of a group. A group contains users who can see and get the app. Thanks for your feedback I updated!